docs/content/en/docs/architecture/monitoring/_index.md

```diff
@@ -6,24 +6,39 @@ description: >
   How do you know if your cluster is healthy?
 ---
 
-When running applications in Kubernetes observability is key. With K8ssandra and cass-operator, each Apache Cassandra® pod is configured with the DataStax [Metrics Collector for Apache Cassandra](https://github.com/datastax/metric-collector-for-apache-cassandra). This exposes Cassandra node-level metrics in the Prometheus format covering everything from operations per second and latency to compaction throughput and heap usage.
+When running applications in Kubernetes, observability is key. K8ssandra includes Prometheus and Grafana for the storage and visualization of metrics associated with the Cassandra cluster.
+
+![Monitoring Overview](monitoring-overview.png)
+
+Cassandra node-level metrics are reported in the Prometheus format, covering everything from operations per second and latency, to compaction throughput and heap usage. Examples of these metrics are shown in the Grafana dashboard below.
 
 ![Grafana Overview](grafana-overview.png)
 
-K8ssandra goes a step further providing a deployment of Prometheus and Grafana for the storage and visualization of these metrics including:
+## Architecture details
+
+K8ssandra uses the [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack), a Helm chart from the [Prometheus Community](https://prometheus.io/community/) project, to deploy Prometheus and Grafana and connect them to Cassandra, as shown in the figure below.
+
+![Monitoring Architecture](monitoring-architecture.png)
+
+Let's walk through this architecture from left to right. We'll provide links to the Kubernetes documentation so you can dig into those concepts more if you'd like to.
 
-* Prometheus `ServiceMonitor` resource with Metric Relabeling directed at the
-  K8ssandra cluster's Service
-* Grafana `DataSource` resource configured to reference the deployed Prometheus
-  instance
-* Grafana `Dashboard` resources
+* The Cassandra nodes in a K8ssandra-managed cluster are organized in one or more data centers, each of which is represented as a Kubernetes [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/). (We'll focus here on details of the Cassandra node related to monitoring, you can see other details about Cassandra nodes such as how storage is managed on the [Cassandra architecture](/docs/architecture/cassandra) page.)
```

Comment:
Each _rack_ is a stateful set. A 3 rack DC will be comprised of 3 stateful sets. Also, "data center" is considered a single word "datacenter".

Reply:
```suggestion
* The Cassandra nodes in a K8ssandra-managed cluster are organized in one or more datacenters, each of which is composed of one or more racks. Each rack represents a failure domain with replicas being placed across multiple racks (if present). In Kubernetes, racks are represented as [StatefulSets](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/). (We'll focus here on details of the Cassandra node related to monitoring, you can see other details about Cassandra nodes such as how storage is managed on the [Cassandra architecture](/docs/architecture/cassandra) page.)
```